4-tools/spyderexample.py

```diff
@@ -1,28 +1,19 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 """
-Created on Mon Nov  9 18:03:24 2020
+Created on Thu Nov  4 15:39:32 2021
 
-@author: danielmikkola
+@author: Simon
 """
-import numpy as np
-
 @profile
-def movmean(xdata, ydata, window):
-    ydata_new = np.zeros(len(ydata))
-    xdata_new = np.zeros(len(xdata))
-    k = int(window/2)
-    for i in range(len(ydata)):
-        if i < window:
-            ydata_new[i] = np.mean(ydata[:(i+k)])
-            xdata_new[i] = np.mean(xdata[:(i+k)])
-        elif i > len(ydata)-window:
-            ydata_new[i] = np.mean(ydata[(i-k):])
-            xdata_new[i] = np.mean(xdata[(i-k):])
-        else:
-            ydata_new[i] = np.mean(ydata[(i-k):(i+k)])
-            xdata_new[i] = np.mean(xdata[(i-k):(i+k)])
-    return(xdata_new, ydata_new)
+def sieve(n):
+    primes=[]
+    test=list(range(3,n+1, 2) )
+    while test[0]<=n**0.5:
```

Comment:
Why is this not PEP 8 compliant?

Reply:
In order to demonstrate pycodestyle_magic on it later in the presentation.

Reply:
The file should be renamed then, it is not obvious that it will be used for anything other than demonstrating Spyder.

Reply:
Done